src/OpenSage.Game/Audio/AudioBuffer.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenSage.Content;
+using OpenSage.Data.Wav;
+using OpenAL;
+
+namespace OpenSage.Audio
+{
+    public sealed class AudioBuffer : DisposableBase
+    {
+        private uint _handle;
+        private List<AudioSource> _usageList;
```

Comment:
I don't think this usage list is necessary, assuming that `AudioBuffer` instances always live longer than the `AudioSource` that use them, as I think they should.